cpp/models/abm/location.h

```diff
@@ -245,6 +246,21 @@ class Location
         m_capacity_adapted_transmission_risk = consider_capacity;
     }
 
+    /**
+     * Save the current number of individuals in the each infection state at a specific time step
+     * @param t the time step to save
+    */
+    void add_subpopulations_to_time_series(const TimePoint& t);
```

Comment:
```suggestion
    void store_current_population(const TimePoint& t);
```
I would like to name this `store_current_population`. What do you think?

Reply:
Yes. The name is updated.